spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -226,15 +226,15 @@
       it 'adds proper identifiers section' do
         pub_ids = result['identifier']
         # Check the PMID, e.g.
-        # {"type"=>"PMID", "id"=>"23684686", "url"=>"http://www.ncbi.nlm.nih.gov/pubmed/23684686"}
+        # {"type"=>"PMID", "id"=>"23684686", "url"=>"https://www.ncbi.nlm.nih.gov/pubmed/23684686"}
         pmid_hash = pub_ids.find {|id| id['type'] == 'PMID'}
         expect(pmid_hash).to be_instance_of Hash
         pmid = request_data[:pmid]
         expect(pmid_hash['type']).to eq('PMID')
         expect(pmid_hash['id']).to eq(pmid)
-        expect(pmid_hash['url']).to eq("http://www.ncbi.nlm.nih.gov/pubmed/#{pmid}")
+        expect(pmid_hash['url']).to eq("https://www.ncbi.nlm.nih.gov/pubmed/#{pmid}")
         # Check the SULPubId, e.g.
-        # {:type=>"SULPubId", :id=>"2355", :url=>"http://sulcap.stanford.edu/publications/2355"}],
+        # {:type=>"SULPubId", :id=>"2355", :url=>"https://sulcap.stanford.edu/publications/2355"}],
```

Comment:
This is a sufficient change to the specs.  General question - should these specs be checking that the URLs are working by issuing a GET that returns a 200 response?  (This might be a new issue for the backlog.)